playground/AzureFunctionsEndToEnd/AzureFunctionsEndToEnd.ApiService/Program.cs

```diff
@@ -41,7 +41,7 @@ static string RandomString(int length)
 
 app.MapGet("/publish/blob", async (BlobServiceClient client, CancellationToken cancellationToken, int length = 20) =>
 {
-    var container = client.GetBlobContainerClient("blobs");
+    var container = client.GetBlobContainerClient("myblobcontainer");
```

Comment:
At some point, we update this playground app the use the `AddBlobContainer` API but never updated the container names that were used.